Library/Homebrew/test/Rakefile

```diff
@@ -1,8 +1,14 @@
 require "rake"
 require "rake/testtask"
 
+def mac?
+  /darwin/i === RUBY_PLATFORM && !ENV["HOMEBREW_TEST_GENERIC_OS"]
```

Comment:
RuboCop strongly objects to using the “case equality operator” outside of a `case` statement and I agree with the reasoning (though that's quite a widespread pattern in the Homebrew code base). This should work equally well and is clearer:

```ruby
/darwin/i =~ RUBY_PLATFORM && !ENV["HOMEBREW_TEST_GENERIC_OS"]
```